lib/src/components/NxToast/types.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { HTMLAttributes, ReactNode, ReactElement } from 'react';
+import * as PropTypes from 'prop-types';
+
+export interface NxToastProps extends HTMLAttributes<HTMLDivElement> {
+  onClose: () => void;
+  children: ReactElement;
+}
+
+export const nxToastPropTypes: PropTypes.ValidationMap<NxToastProps> = {
+  onClose: PropTypes.func.isRequired,
+  children: PropTypes.any.isRequired
+};
+
+export type NxToastContainerContextType = {
+  onToastClosing: (toast: HTMLElement | null)=> void;
+};
+
+//NxToastContainer
+export type NxToastContainerProps = {
```

Comment:
Let's stick to making these `interface`s and not `type`s

Reply:
> Let's stick to making these `interface`s and not `type`s

Replaced NxToastContainerProps as an interface instead of type here: [f7b84cf](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/f7b84cfcdd548da11902bd465f8625d8e3f96504)